sys/include/memarray.h

```diff
@@ -0,0 +1,55 @@
+/*
+ * Copyright (C) 2017 Tobias Heider <heidert@nm.ifi.lmu.de>
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+#ifndef MEMARRAY_H
+#define MEMARRAY_H
+
+#include "stdint.h"
+#include "stdlib.h"
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+/**
+ * @defgroup    sys_memarray memory array allocator
```

Comment:
please move this comment block up, directly below the copy right statement - just for consistency with RIOT codebase.